README.md

```diff
@@ -57,6 +57,11 @@ When generating `wheel` or `sdist` archives for the `spam` project through Poetr
 `ham` project were declared as `ham = "1.2.3"`.  As a result, package metadata in archives for `spam` will shift from
 `Requires-Dist: ham @ ../ham` to `Requires-Dist: ham (==1.2.3)`
 
+Additionally, to address [path dependencies](https://python-poetry.org/docs/dependency-specification/#path-dependencies) not being portable, this plugin provides
```

Comment:
A: I'd also update the info at the top about automatically adding the functionality with `enable = true` to also list the export command.